### PR TITLE
Make all boto3 calls async

### PR DESCRIPTION
**Pull request by Phuc-Hong-Tran**
For asynchronous code execution

### CODE REVIEW

File: tests/exports/test_r2_bucket.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        with patch.object(settings, 's3_endpoint_url', 'https://17399e149aeaa08c0c7bbb15382fa5c3.r2.cloudflarestorage.com'):
+            with patch.object(settings, 's3_bucket_name', 'r2-dev'):
+                save_response = await write_stat_set_to_r2(power, '/test.json')
+        
+        assert save_response is not None
+        assert isinstance(save_response, int)
+
+    @pytest.mark.asyncio
+    async def test_write_content_to_r2(self):
+        file = open('/home/hong-phuc/Desktop/opennem/tests/fixtures/test.txt', 'r')
```
Comment (nc9) [inline tests/exports/test_r2_bucket.py:26]: make this relative to the project :)

File: tests/exports/test_r2_bucket.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+from opennem import settings
+from opennem.utils.tests import TEST_FIXTURE_PATH
+
+class TestR2Bucket:
+
+    @pytest.mark.asyncio
+    async def test_write_stat_set_to_r2(self):
+        power=load_opennem_dataset_from_file(TEST_FIXTURE_PATH / f"nem_nsw1_7d.json")
+        
+        with patch.object(settings, 's3_endpoint_url', 'https://17399e149aeaa08c0c7bbb15382fa5c3.r2.cloudflarestorage.com'):
```
Comment (nc9) [inline tests/exports/test_r2_bucket.py:17]: is this a new bucket or one created for tests?

File: tests/exports/test_r2_bucket.py (hunk 1 above)
Comment (Phuc-Hong-Tran) [reply tests/exports/test_r2_bucket.py:26]: Will do!

File: tests/exports/test_r2_bucket.py (hunk 2 above)
Comment (Phuc-Hong-Tran) [reply tests/exports/test_r2_bucket.py:17]: This one is the one I created for testing

**Review by nc9 — APPROVED** [review]

**Pull request merged**